app/controllers/homes_controller.rb

```diff
@@ -0,0 +1,12 @@
+class HomesController < ApplicationController
+  def index
+    @google_maps_url = 'https://www.google.com/maps/embed/v1/place?key=' + ENV["GOOGLE_MAPS_API_KEY"] + '&q=Chinatown+Boston+MA'
```

Comment:
Line is too long. [128/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.